### PR TITLE
Separated mobile metrics from browser metrics

### PR DESCRIPTION
**Pull request by LeviSklaroff**
**Issue #:**

**Description of changes:**

**Testing:**

*Can these tested using a demo application? Please provide reproducible step-by-step instructions.*

**Checklist:**

1. Have you successfully run `npm run build:release` locally?

2. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved?

3. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved?

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by devalevenkatesh — DISMISSED** [review]

**Review by ltrung — APPROVED** [review]

File: integration/js/utils/SdkBaseTest.js [hunk 1]
```diff
@@ -34,7 +34,12 @@ class SdkBaseTest extends KiteBaseTest {
     this.originalURL = this.url;
     this.testReady = false;
     this.testFinish = false;
-    this.testName = testName;
+    if(kiteConfig.capabilities.platform == 'IOS' || kiteConfig.capabilities.platform == 'ANDROID') {
```
Comment (simmkyu) [inline integration/js/utils/SdkBaseTest.js:37]: nit
- Use `===`
- `if(` => `if ( `

File: integration/js/utils/SdkBaseTest.js [hunk 2]
```diff
@@ -34,7 +34,12 @@ class SdkBaseTest extends KiteBaseTest {
...
+      this.testName = 'Mobile_'+testName;
```
Comment (simmkyu) [inline integration/js/utils/SdkBaseTest.js:38]: nit: `'+testName` => `' + testName`

File: integration/js/utils/SdkBaseTest.js [hunk 3]
```diff
@@ -34,7 +34,12 @@ class SdkBaseTest extends KiteBaseTest {
...
+    }
```
Comment (simmkyu) [inline integration/js/utils/SdkBaseTest.js:39]: `} else {`

**Review by simmkyu — CHANGES_REQUESTED** [review]

File: integration/js/utils/SdkBaseTest.js (hunk 1 above)
Comment (LeviSklaroff) [reply integration/js/utils/SdkBaseTest.js]: Updated, will make change on 3x to.

**Review by simmkyu — APPROVED** [review]

**Pull request merged**